src/util.test.ts

```diff
@@ -498,14 +498,14 @@ describe('util', () => {
       ).toThrow('Invalid symbol: not a string.');
     });
 
-    it('should throw if symbol is more than 6 characters long', () => {
+    it('should throw if symbol is more than 11 characters long', () => {
```

Comment:
nit: could we retain this old test case using `TKNTKNTKN` as the symbol, and update the outcome to prove that an error is not thrown? then we can add a new test case for `TKNTKNTKNTKN`, just to cover both bases

Reply:
Sure! We can do that.

Reply:
Actually, instead I'll ensure the a symbol at the maximum length is still accepted, to ensure we're checking all boundary conditions. So length 11 rather than 6.

Reply:
sgtm!

Reply:
Additional tests added in 5ba4b25f6f047004e92c486fd092509d9eb1f0e8